ethers-solc/src/lib.rs

```diff
@@ -24,7 +24,7 @@ pub use compile::{
 };
 
 mod config;
-pub use config::{AllowedLibPaths, PathStyle, ProjectPathsConfig, SolcConfig};
+pub use config::{AllowedLibPaths, PathStyle, ProjectPaths, ProjectPathsConfig, SolcConfig};
```

Comment:
ProjectPaths is referenced and exposed in `ProjectPathsConfig`, but not exported at the crate level, so it failed to document